apps/prairielearn/src/pages/instructorAssessmentDownloads/instructorAssessmentDownloads.sql

```diff
@@ -303,257 +323,6 @@ ORDER BY
   variant_number,
   date;
 
--- BLOCK files_for_manual_grading
-WITH
-  final_assessment_instances AS (
-    SELECT DISTINCT
-      ON (g.id, u.user_id) u.user_id,
-      g.id AS group_id,
-      ai.id,
-      g.name AS group_name
-    FROM
-      assessment_instances AS ai
-      LEFT JOIN groups AS g ON (g.id = ai.group_id)
-      LEFT JOIN users AS u ON (u.user_id = ai.user_id)
-    WHERE
-      ai.assessment_id = $assessment_id
-    ORDER BY
-      g.id,
-      u.user_id,
-      ai.number DESC
-  ),
-  submissions_with_files AS (
-    SELECT DISTINCT
-      ON (ai.id, q.qid) u.uid,
-      u.uin,
-      q.qid,
-      s.id AS submission_id,
-      s.submitted_answer,
-      v.params,
-      ai.group_name
-    FROM
-      submissions AS s
-      JOIN variants AS v ON (v.id = s.variant_id)
-      JOIN instance_questions AS iq ON (iq.id = v.instance_question_id)
-      JOIN final_assessment_instances AS ai ON (ai.id = iq.assessment_instance_id)
-      LEFT JOIN users AS u ON (u.user_id = ai.user_id)
-      JOIN assessment_questions AS aq ON (aq.id = iq.assessment_question_id)
-      JOIN questions AS q ON (q.id = aq.question_id)
-    WHERE
-      (
-        v.params ? 'fileName'
-        AND s.submitted_answer ? 'fileData'
-      )
-      OR (s.submitted_answer ? '_files')
-    ORDER BY
-      ai.id,
-      q.qid,
-      s.date DESC
-  ),
-  all_files AS (
-    SELECT
-      uid,
-      group_name,
-      uin,
-      qid,
-      submission_id,
-      (
-        CASE
-          WHEN submitted_answer ? 'fileData' THEN params ->> 'fileName'
-          WHEN submitted_answer ? '_files' THEN f.file ->> 'name'
-        END
-      ) as filename,
-      (
-        CASE
-          WHEN submitted_answer ? 'fileData' THEN submitted_answer ->> 'fileData'
-          WHEN submitted_answer ? '_files' THEN f.file ->> 'contents'
-        END
-      ) as contents
-    FROM
-      submissions_with_files AS s
-      LEFT JOIN (
-        SELECT
-          submission_id AS id,
-          jsonb_array_elements(submitted_answer -> '_files') AS file
-        FROM
-          submissions_with_files
-        WHERE
-          submitted_answer ? '_files'
-      ) f ON (f.id = submission_id)
-  )
-SELECT
-  (
-    (
-      CASE
-        WHEN $group_work THEN group_name
-        ELSE uid || '_' || uin
-      END
-    ) || '_' || qid || '_' || submission_id || '_' || filename
-  ) AS filename,
-  contents
-FROM
-  all_files
-WHERE
-  filename IS NOT NULL
-  AND contents IS NOT NULL
-ORDER BY
-  (
-    CASE
-      WHEN $group_work THEN group_name
-      ELSE uid
-    END
-  ),
-  (
-    CASE
-      WHEN $group_work THEN NULL
-      ELSE uin
-    END
-  ),
```

Comment:
AFAICT sorting by UIN wasn't doing anything since UIDs are unique and UIN is always the same for a given UID, so I didn't copy this to the updated `submissions_for_manual_grading` query.